CHANGELOG.md

```diff
@@ -6,7 +6,26 @@ accidentally triggering the load of a previous DB version.**
 
 ## Unreleased
 
+## 2.2.0 (2021-04-13)
+
+This release adds major new features since the 2.1.1 release.
+We deem it moderate priority for upgrading.
+
+This release adds the Skip Scan optimization, which significantly 
+improves the performance of queries with DISTINCT ON. This 
+optimization is not available for queries on distributed hypertables.
```

Comment:
```suggestion
improves the select performance of queries on hypertables with
DISTINCT ON. This query optimization is not yet available for
distributed hypertables.
```

Reply:
Does skip scan work on relational tables as well, or just hypertables?

Reply:
It seems to me that it works on plain PG tables too. @svenklemm Am I correct?

Reply:
works on both